### ISSUE TITLE
可否共享模型权重或者给予一定指导

### ISSUE THREAD

**Issue by lzhts1** [YuChuang1205/PAL#4]
您好，我非常感谢您的工作，您的工作给我提供了极大的便利，但是我的复现结果和原论文结果差距有一点大，请问可以提供一定的技术支持或者共享模型权重吗，非常感谢您！

**Comment by lzhts1** [comment]
真的非常非常感谢您的工作，我已经点亮了star

**Comment by YuChuang1205** [comment]
> 您好，我非常感谢您的工作，您的工作给我提供了极大的便利，但是我的复现结果和原论文结果差距有一点大，请问可以提供一定的技术支持或者共享模型权重吗，非常感谢您！

Thank you for your interest in our work. I'm currently preparing for other work, so please forgive me for the slow response. For the PAL framework, according to our experimental results and recent communication with several students, the results are relatively stable. Even some of the reproduced results are better than those in the paper. You can try running the experiment again without changing any settings. Based on feedback from others, you can try lowering the learning rate a bit (5e-4) when experimenting with ALCL-Net. To make it easier for everyone to process, I plan to sort out the relevant weights, which are expected to be released in the next few days. 
   In addition, let me advertise. hhh~~~.  Welcome to learn about another work we published in TGRS2025: https://github.com/YuChuang1205/MSDA-Net

**Comment by YuChuang1205** [comment]
> 您好，我非常感谢您的工作，您的工作给我提供了极大的便利，但是我的复现结果和原论文结果差距有一点大，请问可以提供一定的技术支持或者共享模型权重吗，非常感谢您！

According to the arrangement, it should be completed this week.

**Comment by YuChuang1205** [comment]
I have organized the weights trained on the SIRST3 dataset. The relevant weights can be accessed at the link: https://drive.google.com/file/d/1xrrNpduXsrpStgJ25f9FCPKZYMUOwQA_/view?usp=drive_link

**Comment by YuChuang1205** [comment]
Do you have any further questions about this issue? If not, I will close this issue.

**Comment by lzhts1** [comment]
> Do you have any further questions about this issue? If not, I will close this issue.

我们的复现工作是在IRSTD-1K上出现了与原论文过大的偏差，请问是否可以开源这个数据集的权重呢，特别是ACM和ALCNet的权重

**Comment by YuChuang1205** [comment]
For ACM and ALCNet, these are classic works from a few years ago. Their performance is not very good. Are you talking about IRSTD-1K with single-point supervision or full supervision?

**Comment by lzhts1** [comment]
> For ACM and ALCNet, these are classic works from a few years ago. Their performance is not very good. Are you talking about IRSTD-1K with single-point supervision or full supervision?

Our reproduction work is conducted under the PAL framework with coarse-point supervision, strictly following the configuration you provided, and executed on an A100 GPU.

**Comment by YuChuang1205** [comment]
Because the original network performance of ACM and ALCNet was poor, we did not conduct coarse-point supervision experiments on the separate IRSTD-1k dataset. You can see Table 2 in the manuscript.

**Comment by lzhts1** [comment]
Thank you so much for your help — I’m truly grateful and at a loss for words to express my appreciation. I’ve already starred your project as a token of my thanks!

**Comment by YuChuang1205** [comment]
Thank you for your interest in our work. If you have other questions, please feel free to open additional issues. I will close this issue.
